src/installation/guides/fde.md

```diff
@@ -3,6 +3,13 @@
 **Warning**: Your drive's block device and other information may be different,
 so make sure it is correct.
 
+## Caution
+
+Filesystem-level encryption with AEAD is more reliable than using
+[XTS](https://sockpuppet.org/blog/2014/04/30/you-dont-want-xts/),
+you need to adapt the guide from Alpine Linux wiki for setting up
+[encrypted root on ZFS](https://wiki.alpinelinux.org/wiki/Root_on_ZFS_with_native_encryption)
```

Comment:
I'd rather not point to ZFS on Alpine when https://github.com/zbm-dev/zfsbootmenu/ exists, is a Void first solution, and works superbly well.